src/components/ChatMap/ChatMap.jsx

```diff
@@ -1,47 +1,50 @@
 import React, { useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { MapContainer } from 'react-leaflet/MapContainer';
+import PropTypes from 'prop-types';
+// import { MapContainer } from 'react-leaflet/MapContainer';
 import { GeoJSON } from 'react-leaflet';
 import mapData from '../../data/countries.json';
 import 'leaflet/dist/leaflet.css';
 import '../../css/ChatMap.css';
 
-import { sendDataCountryToBackend } from '../../redux-store/AuthOperations/DataCountryOperation.js';
-import { getUserId } from 'redux-store/AuthOperations/selectors';
-import { CountryName } from './ChatMapStyled';
+import { sendDataCountryToBackend } from '../../redux-store/AuthOperations/AuthOperations.js';
+import {
+  getUserId,
+  getPersistedToken,
+} from 'redux-store/AuthOperations/selectors';
+import { CountryName, ShowCountry, MainMapBlock, MapWrapper } from './ChatMapStyled';
 // import { TileLayer } from 'react-leaflet/TileLayer';
 // import { useMap } from 'react-leaflet/hooks';
 // import { TileLayer, useMap } from 'react-leaflet';
 
-export default function ChatMap() {
+function ChatMap({closeMap}) {
```

Comment:
Let's create components using arrow function approach. 
```suggestion
const ChatMap = ({ closeMap }) => {
```